homeassistant/components/circuit/__init__.py

```diff
@@ -0,0 +1,34 @@
+"""The Unify Circuit component."""
+
+import logging
+
+import voluptuous as vol
+
+from homeassistant.const import CONF_NAME, CONF_URL
+from homeassistant.helpers import config_validation as cv, discovery
+
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = "circuit"
+CONF_WEBHOOK = "webhook"
+
+WEBHOOK_SCHEMA = vol.Schema(
+    {vol.Optional(CONF_NAME): cv.string, vol.Required(CONF_URL): cv.string}
```

Comment:
Where do we use the name item?

Reply:
It's an optional config entry, when it's used we create the notify service with this name. So it's clear in the config which url will be linked to which conversation.